Changelog.md

```diff
@@ -1,5 +1,8 @@
 ### 0.5.7 (unreleased)
 
+Important Bugfixes:
+ * Optimizer: Fix wrong ordering of arguments in byte optimization rule for constants.
```

Comment:
should this get a whole own section "Important Bugfixes" instead of being part of the normal "Bugfixes"?

Reply:
It is in `Important Bugfixes`.

Reply:
That's what I mean, as in, whether it should be ;) I haven't seen that before.

Reply:
Yes, we do that for bugs that go into the bug list.

Reply:
From what I can understand, there is an "Important Bugfixes" Changelog section whenever the said release adds a buglist entry. If this logic holds, then  should we retain it in the changelog for legacy reasons?